### PR TITLE
Dashboard after submission deadline mockup

### PR DESCRIPTION
**Pull request by hammerfunk**
Mocked up the dashboard after submission where the application is unsuccessful, shortlisted and successful.
Check out comments for the conditionals.

The EP award will have a different view: 
- Nominators can't see anything except the submitted awards like the others
- Nominees will get linked directly to the pages via email (since they have no dashboard)

### CODE REVIEW

File: app/assets/stylesheets/frontend/forms.scss [hunk 1]
```diff
@@ -572,6 +572,8 @@ input[type="file"] {
 
 .char-count {
   position: relative;
+}
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Rule declaration should be followed by an empty line

File: app/controllers/content_only_controller.rb [hunk 2]
```diff
@@ -39,10 +39,11 @@ class ContentOnlyController < ApplicationController
...
     @user_award_forms = current_user.account.form_answers.order("award_type")
-    @user_award_forms_trade = @user_award_forms.where("award_type" => "trade")
-    @user_award_forms_innovation = @user_award_forms.where("award_type" => "innovation")
-    @user_award_forms_development = @user_award_forms.where("award_type" => "development")
-    @user_award_forms_promotion = @user_award_forms.where("award_type" => "promotion")
+    @user_award_forms_trade = @user_award_forms.where(award_type: "trade")
+    @user_award_forms_innovation = @user_award_forms.where(award_type: "innovation")
+    @user_award_forms_development = @user_award_forms.where(award_type: "development")
+    @user_award_forms_promotion = @user_award_forms.where(award_type: "promotion")
+    @user_award_forms_submitted = current_user.account.form_answers.where.not(urn: nil).order("award_type")
```
Comment (matthewford) [inline app/controllers/content_only_controller.rb]: Line is too long. [107/100]

File: config/routes.rb [hunk 3]
```diff
@@ -46,6 +46,11 @@
   get "/apply_enterprise_promotion_award"               => "content_only#apply_enterprise_promotion_award",               as: "apply_enterprise_promotion_award"
   get "/award_info_promotion"                           => "content_only#award_info_promotion",                           as: "award_info_promotion"
 
+  get "/declaration_of_corporate_responsibility"        => "content_only#declaration_of_corporate_responsibility",        as: "declaration_of_corporate_responsibility"
```
Comment (matthewford) [inline config/routes.rb]: Line is too long. [167/100]<br>Unnecessary spacing detected.

File: config/routes.rb [hunk 4]
```diff
@@ -46,6 +46,11 @@
...
+  get "/award_winners_section"                          => "content_only#award_winners_section",                          as: "award_winners_section"
```
Comment (matthewford) [inline config/routes.rb]: Line is too long. [149/100]<br>Unnecessary spacing detected.

File: config/routes.rb [hunk 5]
```diff
@@ -46,6 +46,11 @@
...
+  get "/press_comment"                                  => "content_only#press_comment",                                  as: "press_comment"
```
Comment (matthewford) [inline config/routes.rb]: Line is too long. [141/100]<br>Unnecessary spacing detected.

File: config/routes.rb [hunk 6]
```diff
@@ -46,6 +46,11 @@
...
+  get "/buckingham_palace_attendance"                   => "content_only#buckingham_palace_attendance",                   as: "buckingham_palace_attendance"
```
Comment (matthewford) [inline config/routes.rb]: Line is too long. [156/100]<br>Unnecessary spacing detected.

File: app/controllers/content_only_controller.rb (hunk 2 above)
Comment (RuslanHamidullin) [inline app/controllers/content_only_controller.rb]: small correct.
Need to replace this:
```
current_user.account.form_answers.where.not(urn: nil).order("award_type")
```
with
```
current_user.account.form_answers.where(submitted: true).order("award_type")
```

**Comment by RuslanHamidullin** [comment]
Good to merge :+1:

**Pull request merged**